cron/cron.sh

```diff
@@ -9,3 +9,5 @@ do
     echo $line
     go run . --repo=$line --format=csv 2>/dev/null >> $output
```

Comment:
should we build it once, and then just run ./scorecard binary.

Reply:
Fixed up the Dockerfile into a proper multi-stage one.